src/getting_started/installation_options.md

```diff
@@ -0,0 +1,272 @@
+```
+
+You can now use the `parsec-tool` to check that the service is running:
+
+```
+./parsec-tool ping
+```
+
+If the Parsec components are correctly downloaded and running, you should see output similar to the
+following:
+
+```
+[INFO] Service wire protocol version
+1.0
+```
+
+### Controlling the Service Manually
+
+When using the Parsec service as a pre-built download, it will not be installed as a system service.
+Therefore, to stop the service, issue the following command:
+
+```
+pkill parsec
+```
+
+You should see some lines of output ending with:
+
+```
+[INFO parsec] Parsec is now terminated.
+```
+
+You can also cause the service to restart, which can be useful if you have made some configuration
+changes for example. This command will cause the service to reload its configuration and restart:
+
+```
+pkill -SIGHUP parsec
+```
+
+Again, this will produce some lines of output, and the final line will be:
+
+```
+[INFO parsec] Parsec configuration reloaded.
+```
+
+### Running the Test Script
+
+The quick-start bundle also contains the `parsec-cli-tests.sh` testing script, which executes a
+simple set of tests to ensure that the Parsec service is operating correctly. Some of these tests
+use the local `openssl` installation as a point of comparison, ensuring that Parsec's results are
+equivalent to those expected by `openssl`.
+
+As this script uses the `parsec-tool`, the `PARSEC_SERVICE_ENDPOINT` environment variable needs to
+be set as follows:
+
+```
+export PARSEC_SERVICE_ENDPOINT="unix:$(pwd)/parsec.sock"
+```
+
+If parsec-tool is not installed into a directory included in `PATH`, then you also need to define
+`PARSEC_TOOL` environment variable with a full path to it:
+
+```
+export PARSEC_TOOL="$(pwd)/parsec-tool"
+```
+
+To run the script, simply execute it without any arguments as follows:
+
+```
+./parsec-cli-tests.sh
+```
+
+The script will run a sequence of operations and produce output along the following lines:
+
+```
+Checking Parsec service...
+[INFO ] Service wire protocol version
+1.0
+
+Testing Mbed Crypto provider
+
+- Test random number generation
+[DEBUG] Parsec BasicClient created with implicit provider "Mbed Crypto provider" and authentication data "UnixPeerCredentials"
+[INFO ] Generating 10 random bytes...
+[DEBUG] Running getuid
+[INFO ] Random bytes:
+A6 F5 90 24 DF FF 50 1F 29 2E
+....
+```
+
+The `parsec-cli-tests.sh` script also accepts some command-line parameters to adjust its behaviour.
+You can use the `-h` option to get additional help on these.
+
+**Note:** If openssl is not installed into a directory included in `PATH` then you also need to
+define `OPENSSL` environment variable with a full path to it:
+
+```
+export OPENSSL="<full path>/openssl"
+```
+
+## Option 3: Build from Source Code
+
+If the above installation options are not suitable, please refer to the instructions for [building
+from source code](../parsec_service/build_run.md) and [secure installation on
```

Comment:
I would assume that the line breaks after `on` (might be) due to the format restrictions.
However, it feels weird. If the format restrictions are the reason, we might need to revise them.
It's happening in different places, like line `261` and line 269.
ps: (worth sharing): I use vscode to view the output while editing (reviewing). It makes it easier to visualize the final shape

Reply:
The text needs to be passed through vmdfmt with the same formatting rules as the CI checks, otherwise the CI fails.

Reply:
The important thing is whether it renders correctly. If there is a problem with the rendering then we need to fix it, otherwise we have to do whatever the CI `vmdfmt` settings require,